docs/markdown/Using Pants/generating-version-tags.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
It's a bummer that this will be a top-level page in our left nav bar because I suspect many people won't need this topic. But I think it's going to be tough to get better page organization until we have more nesting than Readme gives us.

What do you think about adding something like `(Advanced)` to the title? Some type of signal that most day-to-day users can skip this page, given the feedback it's overwhelming how many docs we have.

Reply:
It seems about the same as "Setting up an IDE" (which I cribbed off). If we want to reorg all that, I think it's a separate topic. 

I don't want to introduce a new "advanced" convention ad-hoc for this page. If we decide we want an "Advanced" page with this and others as a sub-page then we can do that separately?